R/bayesiannetworkanalysis.R

```diff
@@ -0,0 +1,1297 @@
+#
+# Copyright (C) 2018 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+BayesianNetworkAnalysis <- function(jaspResults, dataset, options) {
+  
+  options[["missingValues"]] <- "listwise" # Unfortunately BDgraph does not work with pairwise missing values
```

Comment:
perhaps remove the option from the QML interface?

Reply:
I did remove the option from the QML interface, but I use the function `.networkAnalysisReadData` from the standard network module, which needs a `missingValues` argument in the options. Should I best specify this in a comment, @vandenman?

Reply:
yes, please leave a comment!